codepipeline/main.tf

```diff
@@ -0,0 +1,19 @@
+terraform {
+  required_version = ">= 1.3"
+  required_providers {
+    aws = {
+      source  = "hashicorp/aws"
+      version = "~> 4.0"
+    }
+  }
+
+  backend "s3" {
+    bucket = "doorway-cicd-state"
```

Comment:
Just a quick note that this will need to be changed to the state bucket in doorway before merge

Reply:
Done.